### PR TITLE
Add PagerDuty e2e test and fix dockerfile

### PR DESCRIPTION
**Pull request by karthikperu7**
Testing if the secrets exist in the target cluster after installation.
Also added `RUN go mod vendor` in the `dockerfile` to fix the image build which was failing.

### CODE REVIEW

File: test/operators/pagerduty.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package operators
+
+import (
+	"github.com/onsi/ginkgo"
+	"github.com/openshift/osde2e/pkg/helper"
+)
+
+var _ = ginkgo.Describe("[OSD] Pagerduty Operator", func() {
```
Comment (meowfaceman) [inline test/operators/pagerduty.go]: Could you add the string `[Suite: operators]` to the beginning of this Ginkgo.describe string? This is a brand new thing. If you rebase on master you'll see examples in the other operator tests.

Comment (karthikperu7) [reply test/operators/pagerduty.go]: Cool will do that now! thanks

**Comment by karthikperu7** [comment]
@meowfaceman Added [Suite: operators]

**Pull request merged**